bundle/CMakeLists.txt

```diff
@@ -64,17 +64,22 @@ yoda_add_target_clean_all(
   "${CMAKE_BINARY_DIR}/protobuf-prefix"
 )
 
-if(NOT DEFINED PYTHON_EXECUTABLE)
-  find_package(PythonInterp 3.4 REQUIRED)
-endif()
-
 include("thirdparty/DawnAddProtobuf")
 
 install(DIRECTORY ${PROTOBUF_PYTHON_INSTALL}/google DESTINATION ${CMAKE_INSTALL_PREFIX}/python)
 
 include("thirdparty/DawnAddBoost")
 
-set(dawn_cmake_args -DProtobuf_DIR=${Protobuf_DIR} -DBOOST_ROOT=${BOOST_ROOT} -DDAWN_PYTHON=ON -DDAWN_EXAMPLES=ON)
+option(DAWN_BUNDLE_PYTHON "Build and install the Python module interface to HIR" OFF)
+option(DAWN_BUNDLE_JAVA "Build and install the java interface to HIR" OFF)
+
+if( DAWN_BUNDLE_PYTHON )
+  if(NOT DEFINED PYTHON_EXECUTABLE)
+    find_package(PythonInterp 3.4 REQUIRED)
+  endif()
+endif()
+
+set(dawn_cmake_args -DProtobuf_DIR=${Protobuf_DIR} -DBOOST_ROOT=${BOOST_ROOT} -DDAWN_PYTHON=${DAWN_BUNDLE_PYTHON} -DDAWN_EXAMPLES=ON -DDAWN_JAVA=${DAWN_BUNDLE_JAVA})

```

Comment:
Is the plan to extend the dawn examples so there is no hard requirement between examples and python? I think we can leave it like that. Just curious on what you think where this should go?